### DISCUSSION TITLE
What means this warning on Vercel: "WARNING: Unable to find source file for page

### DISCUSSION THREAD

**Discussion by MoOx** [vercel/next.js#23999] [Help]
I am new to next.js and Vercel, and when my deploy is finished, I can see some warnings:

<img width="1056" alt="image" src="https://user-images.githubusercontent.com/157534/114433335-fcf77000-9bc1-11eb-8231-1bd00ddb490b.png">

```
18:58:48.136 | WARNING: Unable to find source file for page [pageOrCategorySlug].js
18:58:48.137 | WARNING: Unable to find source file for page [pageOrCategorySlug]/after/[cursorAfter].js
18:58:48.138 | WARNING: Unable to find source file for page [pageOrCategorySlug]/before/[cursorBefore].js
18:58:48.138 | WARNING: Unable to find source file for page after/[cursorAfter].js
18:58:48.139 | WARNING: Unable to find source file for page before/[cursorBefore].js
18:58:48.140 | WARNING: Unable to find source file for page index.js
```

Any idea what they mean?

**Comment by ijjk** [comment]
Hi, can you provide a minimal reproduction where you are seeing the above warnings? These should not be shown on the latest Version of Next.js `v10.1.3`

These are shown when we aren't able to match an output file to the source page file since the source file is used when applying certain configurations for serverless functions on Vercel

**Reply by MoOx** [reply]
I am using 10.0.8, I guess I will update and see if warnings goes away.

**Reply by ijjk** [reply]
If you're still seeing the errors after updating, please share a minimal repo and we can investigate further!

**Reply by MoOx** [reply]
09:48:18.641 | Detected Next.js version: 10.1.3

Still getting this warnings. You can try building this commit https://github.com/MoOx/dame.bio/commit/5fb8d5db0a0191c81251f3a4055ff694ecc40a7d
